sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -19,6 +19,11 @@
 import sys
 
 
+# Contains a stack of handler functions.
```

Comment:
Could we add more comments in the codes to explain the decorator handlers.
To my understanding, it is a placeholder in the SDK such that the compiler could inject the any codes it wants to run.

Reply:
Sure. I can add a comment explaining it in more details.

Reply:
Added a better description.